src/openforms/forms/models/form.py

```diff
@@ -345,6 +346,16 @@ def get_authentication_backends_display(self):
         "authentication backend(s)"
     )
 
+    def get_cosign_component(self) -> JSONObject | None:
```

Comment:
perhaps we can define a more strict `TypedDict` for the co-sign component shape in `openforms.formio.components.custom`, similar to how we have `openforms.formio.components.vanilla.FileComponent` to accurately reflect the shape of the object returned?